test/index.test.js

```diff
@@ -0,0 +1,65 @@
+import Gameboard from '../src/modules/board';
+import Player from '../src/modules/player';
```

Comment:
[Optional]

Why not separate the tests into two files? This way you have a test file for `Gameboard` and `Player` separately.